bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/commonChannels.xml

```diff
@@ -268,12 +277,17 @@
 	<channel-type id="usedhours">
 		<item-type>Number</item-type>
 		<label>Run Time</label>
-		<state pattern="%.0f" readOnly="true"/>
+		<state pattern="%.0f h" readOnly="true"/>
+	</channel-type>
+	<channel-type id="powerhours">
+		<item-type>Number</item-type>
```

Comment:
similar as previous one
Please make this a Number:Time with unit: hours

Reply:
changed in next commit.